myapp/app/views/tasks/index.html.erb

```diff
@@ -3,23 +3,20 @@
 
   <%= link_to 'New Task', new_task_path, class: 'btn btn-primary mb-3' %>
 
-  <table class="table table-striped">
-    <thead>
-      <tr>
-        <th>Title</th>
-        <th>Description</th>
-        <th colspan="2"></th>
-      </tr>
-    </thead>
-    <tbody>
-      <% @tasks.each do |task| %>
-        <tr>
-          <td><%= link_to task.title, task %></td>
-          <td><%= task.description %></td>
-          <td><%= link_to 'Edit', edit_task_path(task), class: 'btn btn-warning btn-sm' %></td>
-          <td><%= button_to 'Delete', task, method: :delete, class: 'btn btn-danger btn-sm', data: { confirm: 'Are you sure?' } %></td>
-        </tr>
-      <% end %>
-    </tbody>
-  </table>
+  <div class="row">
+    <% @tasks.each do |task| %>
+      <div class="col-md-3"> <!-- 调整列的宽度 -->
```

Comment:
FYI: I'd prefer you to comment in English, normally. But that's OK, because this is an training project.